src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -649,6 +656,9 @@ private bool IsInScope(ITypeParameterSymbol typeParameterSymbol)
 
         private static class Parser
         {
+            private static readonly SymbolEqualityComparer s_symbolEqualityComparer =
```

Comment:
We should define this as internal on `SymbolEqualityComparer`, rather than just here.